src/lib/data/store-types.js

```diff
@@ -147,5 +161,11 @@ export const groupStore = (/**@type{any}*/ groupType, /**@type{any}*/ props) =>
         subs[key].forEach((sub) => sub(vals[key], mods[key]));
     };
 
-    return { subscribe, set };
+    const skip = (val, key = 'default', mod = undefined) => {
+        stores[key].skip(val);
+        mods[key] = mod;
+        subs[key].forEach((sub) => sub(vals[key], mods[key]));
+    }
+
+    return { subscribe, set, skip };
```

Comment:
@FyreByrd I moved the `skip()` functionality to `referenceStore` (so that it can be used by swiping as well), however, I had to patch it through `groupStore` (and notify the subs).  Any thoughts on a better way to do this?

Reply:
I made some changes to generalize the process of exposing extra needed functionality through groupStore and opened a PR into the branch in your repo.